### ISSUE TITLE
Multiplayer game crashing in sandbox mode with <4 characters.

### ISSUE THREAD

**Issue by Kuro-dev** [stiffmeds/Obeliskial-Options#5]
```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Obeliskial_Options.Patches.InitPrefix (EmoteManager& instance) (at <d9d8caa396744193b3052b7dbf48082e>:0)
(wrapper dynamic-method) EmoteManager.DMD<EmoteManager::Init>(EmoteManager)
MatchManager+<BeginMatch>d346.MoveNext () (at <8e4070a00f5b4aeca94fdd980cfe5ab1>:0)
UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at <801ef9485ea54ad392ab1752511dc748>:0)
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
MatchManager:NET_BeginMatch(Int32)
System.Reflection.MethodBase:Invoke(Object, Object[])
Photon.Pun.PhotonNetwork:ExecuteRpc(Hashtable, Player)
Photon.Pun.PhotonNetwork:OnEvent(EventData)
Photon.Realtime.LoadBalancingClient:OnEvent(EventData)
ExitGames.Client.Photon.PeerBase:DeserializeMessageAndCallback(StreamBuffer)
ExitGames.Client.Photon.EnetPeer:DispatchIncomingCommands()
ExitGames.Client.Photon.PhotonPeer:DispatchIncomingCommands()
Photon.Pun.PhotonHandler:Dispatch()
Photon.Pun.PhotonHandler:FixedUpdate()
```

Steps to reproduce: 
1. start a MP game
2. turn on sandbox mode
3. select less characters in the options
4. start the game
5. enter combat
Game will load infinitely, and show the above NPE in console.

**Comment by Kuro-dev** [comment]
Id be happy to help with the debugging process. Just let me know if i can help somehow, im not a CS dev, but since Java is rather similar, i might be able to help.

**Comment by stiffmeds** [comment]
Thanks for the bug report! Should be an easy fix; I'll do it this arvo.

**Comment by stiffmeds** [comment]
(oh, and in the meantime you can mitigate this by disabling the "Emotional" option)

**Comment by stiffmeds** [comment]
Fixed in release [v2.0.5](https://github.com/stiffmeds/Obeliskial-Options/releases/tag/v2.0.5), which has been released on Thunderstore + this repo.

Cheers!
